### DISCUSSION TITLE
What version of Osmand does the test.osmand.net/map/ use?

### DISCUSSION THREAD

**Discussion by Guylby** [osmandapp/OsmAnd#19448] [Q&A] [answered]
Hi,

When I want to test the Osmand routing parameters and route calculation, I find it way easier to do it on the Osmand website. 
Does the test.osmand map version always use the latest nightly build? Or is there a way to know which version is used?

I don't want to make bug reports for things that have already been fixed or from outdated routing version.

Thanks!

**Comment by vshcherb** [comment, marked as answer]
test.osmand.net/map is purely development server and unreliable version. Even nightly version doesn't use it, it's only for testing purposes

**Reply by Guylby** [reply]
Thanks for the answer,

So I assume if I find routing bug on it it's not reliable enough to create bug reports on it?

**Reply by vshcherb** [reply]
Reliable for bugs, we use for testing purposes

**Reply by Guylby** [reply]
Thanks I'll keep it in mind !
